src/main/kotlin/com/vito/sanel/models/MemberType.kt

```diff
@@ -0,0 +1,29 @@
+package com.vito.sanel.models
+
+const val DEFAULT_MAX_BOARD_LENGTH = 50
```

Comment:
Why we use `DEFAULT_MAX_BOARD_LENGTH + 1` below, not `DEFAULT_MAX_BOARD_LENGTH`?